### ISSUE TITLE
U13, U14 pin numbers don't match the hardware

### ISSUE THREAD

**Issue by mlinse** [NextThingCo/PocketCHIP-PCB#3]
As the subject says. For example, U14 pin 1 is GND on the CHIP, and it's shown as 5V on the schematic.
U14 pin 3 is UART1-TX on the CHIP, and it's shown as HPL on the schematic.
There seems to be an odd pin - even pin swap error in the schematic.

**Comment by blinkengus** [comment]
Good catch -- it looks like the device pin numbering in the schematic was even-odd flipped relative to the actual pin numbers. It's harmless, but I agree confusing. Issue has been added to a PocketCHIP punchlist, thanks for reporting it.
